### PR TITLE
feat(network): add Heco

### PR DESCRIPTION
**Pull request by developerfred**
## Description

- [x] add Heco Blockchain
- [x] add Mdex Exchange

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist Contracts

- [ ] RedPacket 
- [x] balanceChecker
- [ ] MaskBox

### CODE REVIEW

File: packages/web3-constants/evm/cryptoartai.json [hunk 1]
```diff
@@ -57,6 +59,7 @@
         "Celo": "",
         "Fantom": "",
         "Aurora": "",
-        "Aurora_Testnet": ""
+        "Aurora_Testnet": "",
+        "Heco": ""
```
Comment (guanbinrui) [inline packages/web3-constants/evm/cryptoartai.json:63]: Debank supports Hecko. Please set a valid value.

File: packages/web3-constants/evm/explorer.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
         "Celo": [],
         "Fantom": ["AST4WWPNEYDURUXG2GH32JZMYWEFDP999S"],
         "Aurora": [],
-        "Aurora_Testnet": []
+        "Aurora_Testnet": [],
+        "Heco": []
```
Comment (guanbinrui) [inline packages/web3-constants/evm/explorer.json]:
```suggestion
        "Heco": ["PUIZXBKP51T1PTRBC465EY662899P7R6WA"]
```

File: packages/web3-constants/evm/explorer.json [hunk 3]
```diff
@@ -37,6 +38,7 @@
         "Celo": "https://explorer.celo.org/api",
         "Fantom": "https://api.ftmscan.com/api",
         "Aurora": "https://explorer.mainnet.aurora.dev/api",
-        "Aurora_Testnet": "https://explorer.testnet.aurora.dev/api"
+        "Aurora_Testnet": "https://explorer.testnet.aurora.dev/api",
+        "Heco": ""
```
Comment (guanbinrui) [inline packages/web3-constants/evm/explorer.json]:
```suggestion
        "Heco": "https://api.hecoinfo.com/api"
```

File: packages/web3-constants/evm/rpc.json [hunk 4]
```diff
@@ -59,7 +59,8 @@
         "Celo": ["https://forno.celo.org"],
         "Fantom": ["https://rpc.ftm.tools/"],
         "Aurora": ["https://mainnet.aurora.dev"],
-        "Aurora_Testnet": ["https://testnet.aurora.dev"]
+        "Aurora_Testnet": ["https://testnet.aurora.dev"],
+        "Heco": []
```
Comment (guanbinrui) [inline packages/web3-constants/evm/rpc.json]:
```suggestion
        "Heco": ["https://http-mainnet-node.huobichain.com/"]
```

File: packages/web3-constants/evm/rpc.json [hunk 5]
```diff
@@ -79,6 +80,7 @@
         "Celo": [0, 0, 0, 0, 0],
         "Fantom": [0, 0, 0, 0, 0],
         "Aurora": [0, 0, 0, 0, 0],
-        "Aurora_Testnet": [0, 0, 0, 0, 0]
+        "Aurora_Testnet": [0, 0, 0, 0, 0],
+        "Heco": []
```
Comment (guanbinrui) [inline packages/web3-constants/evm/rpc.json]:
```suggestion
        "Heco": [0, 0, 0, 0, 0]
```

File: packages/web3-constants/evm/explorer.json (hunk 3 above)
Comment (developerfred) [reply packages/web3-constants/evm/explorer.json]: done

File: packages/web3-constants/evm/explorer.json (hunk 2 above)
Comment (developerfred) [reply packages/web3-constants/evm/explorer.json]: done

**Comment by developerfred** [comment]
- [ ] Qa review @wenluomask

**Review by Jack-Works — APPROVED** [review]

**Comment by guanbinrui** [comment]
I'm closing this. Cause it's not a part of any bounty by now. Please refer to the bounty issue for opening a new PR.

**Pull request closed without merging**